commit 2985d4062ebf4204bbd373456a810d558698f9f5
Author: dtucker@openbsd.org <dtucker@openbsd.org>
Date:   Tue Jul 25 09:22:25 2017 +0000

    upstream commit
    
    Make WinSCP patterns for SSH_OLD_DHGEX more specific to
    exclude WinSCP 5.10.x and up.  bz#2748, from martin at winscp.net, ok djm@
    
    Upstream-ID: 6fd7c32e99af3952db007aa180e73142ddbc741a

diff --git a/compat.c b/compat.c
index 156a5ea8..d82135e2 100644
--- a/compat.c
+++ b/compat.c
@@ -177,9 +177,12 @@ compat_datafellows(const char *version)
 		  "TTSSH/2.72*",	SSH_BUG_HOSTKEYS },
 		{ "WinSCP_release_4*,"
 		  "WinSCP_release_5.0*,"
-		  "WinSCP_release_5.1*,"
-		  "WinSCP_release_5.5*,"
-		  "WinSCP_release_5.6*,"
+		  "WinSCP_release_5.1,"
+		  "WinSCP_release_5.1.*,"
+		  "WinSCP_release_5.5,"
+		  "WinSCP_release_5.5.*,"
+		  "WinSCP_release_5.6,"
+		  "WinSCP_release_5.6.*,"
 		  "WinSCP_release_5.7,"
 		  "WinSCP_release_5.7.1,"
 		  "WinSCP_release_5.7.2,"
